README.markdown

```diff
@@ -22,3 +22,51 @@ long the outcome is distributed under the same license. Otherwise, you must
 obtain a [commercial license](./LICENSE-COMMERCIAL) that removes such
 restrictions. Read more about our licensing approach
 [here](https://www.sourcemeta.com/licensing/).
+
+Example
```

Comment:
Let's put the example right after the project description at the top of the README and we can avoid this extra section